kolibri/core/content/utils/sqlalchemybridge.py

```diff
@@ -413,6 +414,15 @@ def _by_uuids(field, ids, validate, include, vendor=None):
     return UnaryExpression(field, modifier=operators.custom_op(empty_query))
 
 
+checksum_re = re.compile("^[0-9a-e]{32}$")
+
+
+def _validate_checksums(checksums):
+    for checksum in checksums:
+        if not checksum_re.match(checksum):
+            raise ValueError("Invalid checksum")
```

Comment:
Could be handy for debugging if it included the bad checksum in the error message?

Reply:
```suggestion
            raise ValueError("Invalid checksum: {}".format(checksum))
```